src/Stryker.CLI/README.md

```diff
@@ -54,6 +54,53 @@ All available loglevels are:
 
 `dotnet stryker --max-concurrent-test-runners`
 
+#### Mutation testrun reporters 
+
+`dotnet stryker --reporters ['ConsoleReport', 'ConsoleProgressBar']`
+
+All available reporter options are:
+ * All - This will enable all available reporters
```

Comment:
It seems unlogical to start with all. Readers won't know what all reporters are when first reading this.